pages/identities.tsx

```
@@ -109,8 +109,7 @@ const Identities: NextPage = () => {
             <section className="mt-20 w-full">
               <IdentitiesSkeleton />
             </section>
-          ) : ownedIdentities.length + externalDomains.length === 0 ||
-            !address ? (
+          ) : false ? (
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)